wordpress-mobile/WordPress-iOS
# Fixes loading issues in the new Reader

Improves topic syncing in ReaderStreamViewController

Fixes NA
Ref: #13580

To test:

- Go to Reader tab
- Switch between tabs and make sure that the content displays as expected
- log out and log in again (if you have more than one account, use another account)
- Go to reader tab
- make sure that the posts load correctly and that, when switching between tabs, the content displays as expected

PR submission checklist:

- [x] ~~I have considered adding accessibility improvements for my changes.~~ accessibility for the new reader will be handled in a future PR
